src/Raven.Server/Documents/Transformers/CurrentTransformationScope.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Raven.Server.Documents.Indexes.Static;
+
+namespace Raven.Server.Documents.Transformers
+{
+    public class CurrentTransformationScope
+    {
+        private readonly DocumentDatabase _documentDatabase;
+
+        public readonly IndexingFunc TransformResults;
+
+        [ThreadStatic]
+        public static CurrentTransformationScope Current;
```

Comment:
For now it is safe to use it with ThreadStatic. I can change it or monitor this and change it later if necessary. How to handle this?